docs/using-controller-manager-with-kubeadm.md

```diff
@@ -52,8 +52,13 @@
 
 - Create a configmap containing the cloud configuration for cloud-controller-manager.
 
+   Encode your `cloud.conf` file content using base64: `base64 -w 0 cloud.conf`
+
+   Update `cloud.conf` configuration in `manifests/controller-manager/cloud-config-secret.yaml` file
+by using the result of the above command.
+
     ```shell
-    kubectl create configmap cloud-config --from-file=/etc/kubernetes/cloud-config -n kube-system
+    kubectl -f manifests/controller-manager/cloud-config-secret.yaml apply
```

Comment:
Why not including all the steps(base64, configuration update, etc.) above in the shell so it's more convenient for the user?

Reply:
yes

Reply:
Updated